src/dmd/statementsem.d

```diff
@@ -4012,8 +4012,8 @@ private extern (C++) final class StatementSemanticVisitor : Visitor
         tfs.tryBody = sc.tryBody;
 
         auto sc2 = sc.push();
-        sc.tryBody = tfs;
-        tfs._body = tfs._body.statementSemantic(sc);
+        sc2.tryBody = tfs;
+        tfs._body = tfs._body.statementSemantic(sc2);
```

Comment:
Specifically, this change here is in the backtrace.

Reply:
Weird, the only change here is the nested scope instead of the enclosing one... maybe there there is a bug in the symbol/module lookup code that was uncovered by this change.